modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -486,7 +489,9 @@ private TaskProvider<Zip> _baseConfigureClientExtensionProject(
 
 		addTaskDockerDeploy(
 			project, buildClientExtensionZipTaskProvider,
-			new File(workspaceExtension.getDockerDir(), "client-extensions"));
+			new File(
+				workspaceExtension.getDockerDir(),
+				"client-extensions/" + virtualInstanceId));
```

Comment:
For an operation like this, I'd say create a private method that accepts a path string and a project and returns the new path if there's a virtual instance, or just returns the original path if not. Then it could be applied everywhere and you'd only have to retrieve the VID once.

Reply:
I updated the code, can you please check?